### PR TITLE
Update links in winforms/wcf/windows-workflow-foundation folders

### PR DESCRIPTION
**Pull request by gewarren**
- replace redirected links
- replace site-relative links with file-relative links

Contributes to #15560

### CODE REVIEW

**Comment by gewarren** [comment]
@thraka Could you take a look at this one?

**Comment by Thraka** [comment]
Sure

File: docs/framework/windows-workflow-foundation/samples/built-in-activities.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ This section contains samples that demonstrate built-in Windows Workflow Foundat
  [Load From XAML](load-from-xaml.md)  
  Demonstrates how to dynamically load a XAML workflow without having to run the XamlBuildTask tool.
   
- [Using the InvokeMethod Activity](using-the-invokemethod-activity.md)  
+ [Using the InvokeMethod Activity](built-in-activities.md)
```
Comment (Thraka) [inline docs/framework/windows-workflow-foundation/samples/built-in-activities.md]: This just navigates back to the same page. We should remove this link and remove the page from the TOC.

**Review by Thraka — CHANGES_REQUESTED** [review]

**Comment by gewarren** [comment]
@thraka Thanks for the review. I made the requested changes.

**Review by Thraka — APPROVED** [review]
Awesome!

**Pull request merged**